sendinblue.php

```diff
@@ -2605,6 +2669,24 @@ public function subscribeByruntime($email, $post_value = '')
             $attribute_data[] = $value_group;
             $attribute_key[] = 'GROUP_ID';
         }
+        if (version_compare(_PS_VERSION_, '1.5.3.4', '>') && isset($id_shop) && !empty($email)) {
+            if ($id_shop === null) {
+                $id_shop = 1;
+            }
+            $all_group_reg = array();
+            $all_group_unsubs = array();
+            $all_group_reg = Db::getInstance()->getRow('SELECT GROUP_CONCAT(id_shop) as storeid FROM ' . _DB_PREFIX_ . 'customer  WHERE email = "' . pSQL($email) . '" GROUP BY email');
+            
+            $all_group_unsubs = Db::getInstance()->getRow('SELECT GROUP_CONCAT(id_shop) as storeid FROM ' . _DB_PREFIX_ . 'sendin_newsletter  WHERE email = "' . pSQL($email) . '" GROUP BY email');
+            $data_first_store = array();
```

Comment:
Be careful, you initialize multiple times the same variable.

Ther first time with an `array()`, then with a function result.